tzwhere/tzwhere.py

```diff
@@ -167,11 +191,23 @@ def read_tzworld(input_kind='json', path=None):
         return reader(path)
 
     @staticmethod
+    def _open(filename, mode='r'):
+        filename_zipped = filename + tzwhere.ZIPPED
+        if os.path.isfile(filename):
+            return open(filename, mode)
+        elif os.path.isfile(filename_zipped):
+            return closing(gzip.open(filename_zipped, mode))
+        else:
+            url = "{0}/{1}".format(tzwhere.URL_PREFIX, os.path.basename(filename_zipped))
+            content = requests.get(url).content
+            return closing(gzip.GzipFile(fileobj=BytesIO(content)))
```

Comment:
This whole method is unusual to me. There is magic in backing off to a .gz file and also magic in attempting to download the file from the internet.  I'd prefer explicit vs magic.  

`URL_PREFIX` seems a tad static - if the user wants to host the file somewhere, let them pass in a URL and take care of the hosting. Better yet, just accept a file like object and let the user take care of providing the stream - doesn't matter where the file is actually stored.